constants.sh

```diff
@@ -15,6 +15,7 @@ OS=(
 )
 
 DEFAULT_PORT=9090
+DEFAULT_OS_REGISTRY="registry.devshift.net/fabric8-hdd"
```

Comment:
have it as in https://github.com/fabric8-hdd/fabric8-hypothesis-core-image/pull/16/files#diff-3254677a7917c6c01f55212f86c57fbfR1